tests/app/v2/notifications/test_push_notifications.py

```diff
@@ -150,30 +133,17 @@ def test_does_not_accept_extra_fields(
         assert 'Additional properties are not allowed (foo was unexpected)' in resp_json['errors'][0]['message']
 
 
-class TestPushSending:
-    @pytest.fixture(autouse=True)
-    def mobile_app_sids(self, mocker, request):
-        if 'disable_autouse' in request.keywords:
-            for app in MobileAppType.values():
-                mocker.patch.dict(os.environ, {f'{app}_SID': ''})
-            yield
-        else:
-            for app in MobileAppType.values():
-                mocker.patch.dict(os.environ, {f'{app}_SID': f'some_sid_for_{app}'})
-            yield
-
+class TestDeliverPushCelery:
     @pytest.fixture()
-    def vetext_client(self, mocker):
-        client = mocker.Mock(spec=VETextClient)
-        mocker.patch('app.v2.notifications.rest_push.vetext_client', client)
-        return client
+    def deliver_push_celery(self, mocker):
+        mocker.patch('app.v2.notifications.rest_push.deliver_push.delay')
```

Comment:
This fixture is identical to the one your created in a test class in a different file.  That suggests to me that you should move it to /tests/app/v2/notifications/conftest.py to avoid repetition.

Reply:
Since this is only used in two place - I'm hesitant to abstract this out and make the testing harder to follow. This mock is fundamental to how we're testing the code and I'd like it to be obvious to any future devs that this mock is happening.